ghost/src/game/game_base.cpp

```diff
@@ -6561,11 +6561,14 @@ string CBaseGame :: GetColoredName( string defaultname )
 
     for( vector<CBNET *> :: iterator k = m_OHBot->m_BNETs.begin( ); k != m_OHBot->m_BNETs.end( ); ++k )
     {
-	for( vector<permission> :: iterator i = (*k)->m_Permissions.begin( ); i != (*k)->m_Permissions.end( ); ++i )
+    	if( m_OHBot->m_ColoredNameHide )
```

Comment:
The  _if( m_OHBot->m_ColoredNameHide )_ clause should be prior to _line 6562_, because iteration over  m_BNETs is not necessary if coloredNames are disabled globaly.